velocity/src/main/java/io/github/miniplaceholders/expansion/placeholderapi/velocity/VelocityPlugin.java

```diff
@@ -34,6 +37,22 @@ public final class VelocityPlugin {
     @Inject
     private Logger logger;
     private static final UUID NULL_UUID = UUID.randomUUID();
+    private static final PlaceholderAPI instance = PlaceholderAPI.createInstance();
+    private static final ConcurrentMap<Long, PlaceholderAPI> cachedInstances = Maps.newConcurrentMap();
```

Comment:
Why do you use this method instead of creating a ConcurrentHashMap?

Reply:
I just remembered the method from PAPIProxyBridge, no reason versus creating a new ConcurrentHashMap, it does the exact same thing under the hood it seems